web/src/components/organisms/atoms/SideBar.jsx

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link';
+import { HiTemplate, HiOutlineShoppingBag, HiOutlineUsers } from "react-icons/hi";
+
+const Links = [
```

Comment:
variables should be in camelCase
```suggestion
const links = [
```